orchagent/portsorch.cpp

```diff
@@ -2578,7 +2592,17 @@ bool PortsOrch::addVlan(string vlan_alias)
     sai_attribute_t attr;
     attr.id = SAI_VLAN_ATTR_VLAN_ID;
     attr.value.u16 = vlan_id;
-    sai_status_t status = sai_vlan_api->create_vlan(&vlan_oid, gSwitchId, 1, &attr);
+    sai_status_t status = SAI_STATUS_SUCCESS;
```

Comment:
If the default VLAN is in the `m_portList`, there's no need to modify this function.